kv/replica_slice_test.go

```diff
@@ -110,3 +111,22 @@ func TestReplicaSetMoveToFront(t *testing.T) {
 		t.Errorf("expected order %s, got %s", exp, stores)
 	}
 }
+
+func TestReplicaSetRandomOrder(t *testing.T) {
```

Comment:
This is my mistake. I should've called SortRandomOrder instead of MoveToFront. I checked the make test testrace and make check.